### PR TITLE
updated dependencies and packages for react 18

### LINKED ISSUE #93: Migrate app to react 18

### PR DESCRIPTION
**Pull request by roquesrolando**
Here I migrated the app to use React 18 update some packages and dependencies, removed some unused ones that conflicted with the new version of React, update .babelrc to use new version of babel. From babel version 5 and up the packages are called `@babel` instead of `babel`. Changed the render to match the render in safehouse migration to React 18

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,9 +21,9 @@ yarn-error.log*
 /bundle
 /bundle0
 /package-lock.json
+/pnpm-lock.yaml
```
Comment (yacc) [inline .gitignore]: We should always keep the lock file in git, see:
https://pnpm.io/git#:~:text=You%20should%20always%20commit%20the,able%20to%20skip%20package%20resolution

File: package.json [hunk 2]
```diff
@@ -34,36 +34,32 @@
...
     "lodash": "^4.17.21",
     "prop-types": "^15.8.0",
-    "react": "^17.0.2",
+    "react": "^18.2.0",
     "react-bootstrap": "^2.5.0",
     "react-device-detect": "^2.1.2",
-    "react-dom": "^17.0.2",
+    "react-dom": "^18.2.0",
     "react-markdown-it": "^1.0.2",
-    "react-masonry-component": "^6.3.0",
```
Comment (yacc) [inline package.json:42]: +1 for removing unused dependencies

**Review by yacc — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (roquesrolando) [reply .gitignore]: Oh I though since we were ignoring lock.json we should have ignored lock.yaml I will revert it

**Review by sullof — APPROVED** [review]
Can you just remove the comment in App.js?

**Pull request merged**